src/types.cc

```diff
@@ -852,6 +853,38 @@ namespace RimeApiReg{
   }
 }
 
+namespace SwitcherReg {
+  typedef Switcher T;
+
+  an<T> make(Engine *engine) {
```

Comment:
Not sure if the function name "make" is appropriate. It's more like casting `Engine *` to `Switcher *` than "making" a `Switcher *`. Any suggestions?

Reply:
I think "make" is acceptable. It's not very important since the name is not exported.